src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+      src="./images/imac.jpeg" 
+      alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)" 
+      class="card__img img"
+      >
```

Comment:
Do not forget about indent
```suggestion
      <img 
        src="./images/imac.jpeg" 
        alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)" 
        class="card__img img"
      >
```